### PR TITLE
Document `getPretender` and `setPretender` in README

### PR DESCRIPTION
**Pull request by YoranBrondsema**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1941,6 +1942,16 @@ Usage:
...
 ```
 
+### Pretender
+The addon uses [Pretender](https://github.com/pretenderjs/pretender) to mock the requests. It exposes the functions `getPretender` and `setPretender` to respectively get the Pretender server for the current test or set it. For instance,
+
+```javascript
+import { getPretender } from 'ember-data-factory-guy';
+
+// Passthrough 'data:' requests.
+getPretender().get('data:*', getPretender().passthrough);
```
Comment (danielspaniel) [inline README.md]: Thanks for the PR! :)  
Did you forget to use setPretender ?  or maybe can you add an example for that too ?

Comment (YoranBrondsema) [reply README.md]: I didn't have a use case myself for `setPretender` but still wanted to have it mentioned somewhere on the README.

**Comment by danielspaniel** [comment]
thanks for contributing @YoranBrondsema

**Comment by YoranBrondsema** [comment]
Thanks for the quick merge!

**Pull request merged**